tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -2464,6 +2464,12 @@ tacacs/test_authorization.py::test_authorization_tacacs_and_local:
     conditions:
       - https://github.com/sonic-net/sonic-mgmt/issues/11349
 
+tacacs/test_ro_disk.py:
```

Comment:
we actually never hit this issue, but hitting another once with PTF not properly setup. since this issue looks to be a rare race condition, so we are hesitant to take this change.

is this issue still repro'ing on your side? I am not seeing any hits in the latest run anymore.

Reply:
> we actually never hit this issue, but hitting another once with PTF not properly setup. since this issue looks to be a rare race condition, so we are hesitant to take this change.
> 
> is this issue still repro'ing on your side? I am not seeing any hits in the latest run anymore.

Will check and update

Reply:
from the latest update, the issue cannot be reproduced anymore from your side. we will keep this change as it is.